hpe_3par_kubernetes_manager.py

```diff
@@ -25,6 +25,48 @@
 timeout = 180
 
 
+def hpe_create_service_object(yml):
+    try:
+        namespace = globals.namespace
+        resp = k8s_core_v1.create_namespaced_service(namespace=namespace, body=yml)
+        logging.getLogger().debug("Service created. Status is %s" % resp.metadata.name)
+        return resp
+    except client.rest.ApiException as e:
+        logging.getLogger().error("Exception in creating service: %s" % e)
+        raise e
+
+
+def create_service(yml):
+    obj = None
+    with open(yml) as f:
+        elements = list(yaml.safe_load_all(f))
+        for el in elements:
+            logging.getLogger().debug("======== kind :: %s " % str(el.get('kind')))
+            if str(el.get('kind')) == "Service":
+                logging.getLogger().info("Creating Service...")
+                obj = hpe_create_service_object(el)
+                logging.getLogger().info("Service %s created." % obj.metadata.name)
+    return obj
+
+def hpe_delete_service_object_by_name(service_name, namespace):
+    try:
+        secret = k8s_core_v1.delete_namespaced_service(service_name, namespace=namespace)
+    except client.rest.ApiException as e:
+        logging.getLogger().error("Exception while deleting Service :: %s" % e)
+        raise e
+
+def delete_service(name, namespace):
+    try:
+        logging.getLogger().info("\nDeleting Service %s from namespace %s..." % (name,namespace))
+        hpe_delete_service_object_by_name(name, namespace=namespace)
+        flag = check_if_deleted(timeout, name, "Service", namespace=namespace)
+        if flag:

```

Comment:
if not flag:
	print negative logs
	return False
logging.getLogger().info("\nService %s from namespace %s is deleted." % (name, namespace))
return flag